### PR TITLE
Added comment and INTERNAL_IPS.

### PR DESCRIPTION
**Pull request by boco**
Deleted TODO comment.

### CODE REVIEW

File: piplmesh/settings.py [hunk 1]
```diff
@@ -108,6 +108,11 @@
 # URL prefix for internationalization URLs
 I18N_URL = '/i18n/'
 
+#List of internal IPs from which django-pushserver is accessible
```
Comment (mitar) [inline piplmesh/settings.py]: Not really true, only from where _django-pushserver passthrough callbacks_ are accessible.

Also this setting configures other things, so comment should reflect that. Like _This list also configures IPs from which django-pushserver passthrough callbacks are allowed._

Comment (mitar) [inline piplmesh/settings.py]: We are doing space just after the `#`. See how other comments are done.

**Comment by boco** [comment]
Changed comment.

**Comment by mitar** [comment]
Now you have to update `requirements.txt` with new version of django-pushserver so that this setting will have an effect.

**Comment by boco** [comment]
So I just change "django-pushserver==0.1.8" to, let's say, "django-pushserver==0.1.9" ?

**Comment by mitar** [comment]
Yes, I made 0.1.9 version with your changes.

**Comment by boco** [comment]
Updated.

**Pull request merged**